extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -527,26 +523,22 @@ export async function makeHttpRequest<B>(
 		requestUrl = `${account.properties.providerSettings.settings.armResource.endpoint}${path}`;
 	}
 
-	let response: AzureNetworkResponse<B | ErrorResponseBodyWithError> | undefined = undefined;
+	let response: AxiosResponse<B | ErrorResponseBodyWithError> | undefined;
```

Comment:
We shouldn't need `AzureNetworkResponse` anymore, right? Make sure you go check where that's used and clean it up as necessary as well.